Admin/AdminTreeInterface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Admin;
+
+/**
+ * @author Jules Lamur <contact@juleslamur.fr>
+ */
+interface AdminTreeInterface
+{
+    /**
+     * Return the root ancestor or itself if not a child.
```

Comment:
"Returns" with an "s", or move this to the `@return` annotation (I'd go with the latter)